lib/matplotlib/colors.py

```diff
@@ -668,6 +690,7 @@ def _init(self):
         raise NotImplementedError("Abstract class only")
 
     def is_gray(self):
+        """Determine if the color map is grayscale."""
```

Comment:
Language nitpicking:
```suggestion
        """Return whether the color map is grayscale."""
```
"Determine if" is ok, but "Return" is slightly more explicit.